hawc/apps/study/api.py

```diff
@@ -53,6 +57,10 @@ def types(self, request):
         study_types = self.model.STUDY_TYPE_FIELDS
         return Response(study_types)
 
+    def create(self, request):
+        # permissions check not here; see serializer validation
```

Comment:
i like this comment; what do you think?

Reply:
Yeah I think its a good addition, it'll be a helpful reminder if we ever have to troubleshoot it.